lib/elixir/lib/module.ex

```diff
@@ -695,6 +695,16 @@ defmodule Module do
   defp simplify_signature(other, acc) when is_binary(other),  do: autogenerated(acc, :binary)
   defp simplify_signature(_, acc),                            do: autogenerated(acc, :arg)
 
+  defp simplify_struct(module) when is_atom(module) do
```

Comment:
@whatyouhide I renamed it to `simplify_module_name` which is what it's actually doing ("struct" isn't even involved as far as it knows).